src/views/WireframesPage.js

```diff
@@ -86,31 +86,25 @@ export class WireframesPage extends Component {
    * Request the conversion
    */
   toConvertPage() {
-    this.calcTimeEstimate(this.state.selected.length);
+    const { projectID, authType, accessToken, selected, userID } = this.state;
+
+    this.calcTimeEstimate(selected.length);
     this.setState({
       loaderVisible: true,
     });
-    $.ajax({
-      type: "POST",
-      url: `${BACKEND_ADDRESS}/api/v1/projects/${this.state.projectID}/build?authType=${this.state.authType}&accessToken=${this.state.accessToken}`,
-      contentType: "application/json",
-      dataType: "json",
-      async: true,
-      data: JSON.stringify({
-        wireframeList: this.state.selected,
-        userId: this.state.userID,
-      }),
-      success: function (data) {
-        if (data.success) {
-          window.location.href = `${BACKEND_ADDRESS}/api/v1/projects/${this.state.projectID}/download`;
-        }
-        this.setState({ loaderVisible: false });
-      }.bind(this),
-      error: function (xhr, status, err) {
-        console.log("error");
-        this.setState({ loaderVisible: false });
-      }.bind(this),
-    });
+
+    api
+      .post(
+        `${BACKEND_ADDRESS}/api/v1/projects/${projectID}/build?authType=${authType}&accessToken=${accessToken}`,
+        { wireframeList: selected, userId: userID }
+      )
+      .then((res) => {
+        window.location.href = `${BACKEND_ADDRESS}/api/v1/projects/${projectID}/download`;
```

Comment:
Shouldn't we change this to using history push instead? To follow the convention we use in other pages.